elastic/kibana
# [Alerting] Adding feature flag for enabling/disabling rule import and export

Resolves https://github.com/elastic/kibana/issues/100145

## Summary

Added `xpack.alerting.enableImportExport` to control whether exporting of rules is available. Defaults to `false`.

## To Verify

Run this PR branch. Create some rules and connectors. Go to Saved Objects Management UI and see that connectors are available for export but rules are not. Now set `xpack.alerting.enableImportExport` to `true` in your `kibana.dev.yml`. Reload the UI and see that rules are now listed for export in the Saved Objects Management UI.

### Checklist
